utilities/parse_json_webhook_test.go

```diff
@@ -0,0 +1,38 @@
+package utilities
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var reqJSON = map[string]string{
+	"mobilenumber":      "11122223333",
+	"status":            "送信済",
+	"smsid":             "1234567890",
+	"smstext":           "test",
+	"clicks":            "1",
+	"clicks2":           "0",
+	"clicks3":           "0",
+	"clicks4":           "0",
+	"originalurl":       "http://google.com",
+	"originalurl2":      "",
+	"originalurl3":      "",
+	"originalurl4":      "",
+	"type":              "sms",
+	"returnsms":         "1",
+	"waitreturnsms":     "1",
+	"returnsmsdatetime": "2020年09月01日 10:00:00",
```

Comment:
Wonder if the parser should be responsible for changing this into something more generic (e.g. `2020-09-01 10:00:00`)?

Reply:
@rdehaas-comvex Do you think we should translate the Status to English, or should we let that be done on the SMS-API side? Potentially, the user would actually want it in Japanese I guess. Although I suspect we will be the only users of this library.

Reply:
Good point, `status` could possibly stay the same 🤔 
Unless we want the whole library to be 100% English for consistency.

Reply:
Good point, `status` could possibly stay the same 🤔 
Unless we want the whole library to be 100% English for consistency.

Reply:
I think it's better we make the library 100% english